ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.16.yaml

```diff
@@ -85,11 +85,11 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe-2
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private
-- as: installer-rehearse-vsphere
+- as: installer-rehearse-vsphere-transit-masq
   cron: '@yearly'
   steps:
     cluster_profile: vsphere-connected-2
-    workflow: cucushift-installer-rehearse-vsphere-ipi-customized-resource
+    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet
```

Comment:
this job only for installation, Guess we need to do regression testing after cluster is ready?

Reply:
@zhaozhanqi Yes. We need to run all tests

Reply:
@zhaozhanqi @kuiwang02 We need to merge this worflow asap due to its long overdue. Please leave your comments and suggest changes asap and help merging it fast.

Reply:
if so, we need to add job in `ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml` instead of `ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.16.yaml`

Reply:
@anuragthehatter actually no need me to review it because it has nothing with golang step.
it seem you need installation team and qe-productivity team to review it.

Reply:
@zhaozhanqi I am proposing this to go under  
`ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml` Let me know
```
- as: vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-f7
  cron: 36 13 1,17 * *
  steps:
    cluster_profile: vsphere-2
    test:
    - chain: cucushift-installer-check-cluster-health
    - ref: idp-htpasswd
    - ref: cucushift-pre
    - ref: openshift-extended-test
    - ref: cucushift-e2e
    - ref: openshift-e2e-test-qe-report
    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet
```
Please suggest cron values as well

Reply:
1.  `chain: cucushift-installer-check-cluster-health` and `- ref: idp-htpasswd` should be included in install provision chain
2.  and the test we can directly use existing test chain `openshift-e2e-test-qe-automated-release` 
```
    - ref: cucushift-pre
    - ref: openshift-extended-test
    - ref: cucushift-e2e
    - ref: openshift-e2e-test-qe-report
```

3. Usually we have 2 job, one is for non-destructive cases.  so we can set `f28` instead of `f7` consider the vsphere resource.  

so config should be like "
```
- as: vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-f28
  cron: 36 13 1,17 * *
  steps:
    cluster_profile: vsphere-2
    test:
    - chain: openshift-e2e-test-qe-automated-release
    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet

- as: vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet-f28-destructive
  cron: 36 13 1,17 * *
  steps:
    cluster_profile: vsphere-2
    test:
    - chain: openshift-e2e-test-qe-destructive
    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-custom-transit-switch-masquerade-subnet
```
note. please update both two cron values, you can use the script `./ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh` to generate it.

Reply:
Thanks @zhaozhanqi PTAL